deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -64,21 +70,36 @@ class TrainTestPredictionDrift(TrainTestCheck, ReduceMixin):
     categorical_drift_method: str, default: "cramer_v"
         decides which method to use on categorical variables. Possible values are:
         "cramers_v" for Cramer's V, "PSI" for Population Stability Index (PSI).
+    aggregation_method: str, default: "max"
+        argument for the reduce_output functionality, decides how to aggregate the drift scores of different classes
+        into a single score, when drift is computed on the class probabilities. Possible values are:
+        'weighted': Weighted mean based on the number of samples in each class.
+        'mean': Mean of all drift scores.
+        'none': No averaging. Return a dict with a drift score for each class.
+        'max': Maximum of all the class drift scores.
```

Comment:
Put the default option first in docstring.
Also, "none" is again confusing as a string. Maybe another word?

Reply:
Keeping it for compliance with the feature drift. Also, don't have a really better idea ('all'?)

Reply:
ok.